libraries/AC_WPNav/AC_WPNav.cpp

```diff
@@ -341,9 +341,9 @@ bool AC_WPNav::set_wp_destination_NEU_cm(const Vector3f& destination_neu_cm, boo
                                          _pos_control.get_max_speed_NE_cms(), _pos_control.get_max_speed_up_cms(), _pos_control.get_max_speed_down_cms(),
                                          get_wp_acceleration_cmss(), _wp_accel_z_cmss,
                                          _scurve_snap_max_mssss * 100.0f, _scurve_jerk_max_msss * 100.0f);
-        if (!is_zero(origin_speed)) {
+        if (!is_zero(origin_speed_cms)) {
```

Comment:
it's a little tricky that the origin_speed rename to origin_speed_cms is in a different commit..